src/main/steps/common/task-list/components/notification-banner/utils.test.ts

```diff
@@ -0,0 +1,27 @@
+import { PartyType, State } from '../../../../../app/case/definition';
+
+import { getNotificationBannerConfig } from './utils';
+
+describe('testcase for notification Banner', () => {
+  test('when casetype not mentioned', () => {
+    const data = {
+      id: '12',
+      state: State.SUBMITTED_NOT_PAID,
+    };
+    const party = PartyType.APPLICANT;
+    const language = 'cy';
+
+    expect(getNotificationBannerConfig(data, party, language)).toStrictEqual([]);
+  });
+  // test('when casetype c100', () => {
```

Comment:
@sealDebolina Why this test case is commented out ?
We need to test cases for success scenario as well, please add them